pkg/apis/policy/types.go

```diff
@@ -37,6 +37,16 @@ type PodDisruptionBudgetSpec struct {
 	Selector *unversioned.LabelSelector `json:"selector,omitempty"`
 }
 
+// DisruptedPod contains an information about a pod that was disrupted
+// recently. It is assumed that the pod is in the same namespace as PodDisruptionBudget.
+type DisruptedPod struct {
+	// Name of the pod that will be disrupted.
+	Name string `json:"name,omitempty"`
+
+	// DisruptionTime is the time of when the disruption was planned.
```

Comment:
"planned" is a bit ambiguous -- I assume it's the time the API server processed the eviction request, so maybe say that?